cupy/cuda/cupy_nccl.h

```diff
@@ -193,4 +200,18 @@ ncclResult_t _ncclAllGather(const void* sendbuff, void* recvbuff, size_t sendcou
 #endif // #if (NCCL_VERSION_CODE < 2000)
 }
 
+ncclResult_t _ncclCommGetAsyncError(ncclComm_t comm, ncclResult_t *asyncError) {
```

Comment:
* I think it's better to add stub function using `#ifdef` for newly-introduced APIs, rather than adding `_`-suffixed wrapper functions (which intends to conceal breaking changes of API with the same name between NCCL v1/v2). c.f. https://github.com/cupy/cupy/blob/48220074f2d42007754876446cfecea8c6c628e8/cupy/cuda/cupy_nccl.h#L144-L151
* It seems it's returning `ncclSuccess`. Does it match with what you initially expected? (`When built with old libraries it throws Exceptions.`)

Reply:
- Removed _-prefixed wrapper.
- For ncclGetAsyncError I don't have good idea other than returning that code like other stubs  are doing. Also there aren't suitable error codes in nccl.h IMO.